Source/ThirdParty/ANGLE/src/compiler/translator/tree_util/IntermTraverse.cpp

```diff
@@ -372,6 +372,7 @@ void TLValueTrackingTraverser::traverseBinary(TIntermBinary *node)
             // hasn't been cancelled yet.
             if (postVisit)
                 visit = node->visit(PostVisit, this);
```

Comment:
I think here the assignment would be omitted, and if the return value is nodiscard, (void) would be added to the call site.